.gitlab/repository.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "terraform-aws-terraform-state-output",
+  "url": "https://gitlab.com/infraprints/terraform-aws-terraform-state-output",
```

Use modules concept instead

```suggestion
  "url": "https://gitlab.com/infraprints/modules/terraform-aws-terraform-state-output",
```